src/lib/cache/subCache.cpp

```diff
@@ -682,7 +682,6 @@ void subCacheItemInsert
   const std::string&            georel
 )
 {
-  LM_W(("KZ: inserting sub in cache. extended == %s", httpInfo.extended? "true" : "false"));
```

Comment:
Actually, that's what I did for this PR. I just might have removed a few originating from develop.
No 'bad' LM_W remaining. That said, always good to check again. Doesn't cost more than a few minutes.